api/lib/spree/api/testing_support/v2/platform_contexts.rb

```diff
@@ -87,6 +87,12 @@ def json_api_filter_parameter(example = '')
   end
 end
 
+shared_examples 'no content' do
+  response '204', 'Record updated - no content returned' do
```

Comment:
@MatthewKennedy we should always return the updated record in the response

Reply:
I'll sort this.